### PR TITLE
fix: tslint -> eslint

### PR DESCRIPTION
**Pull request by WillieRuemmele**
### What does this PR do?
adds the default dev-config and dev-scripts to the template

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -2,7 +2,21 @@
 <project version="4">
```
Comment (amphro) [inline .idea/workspace.xml:2]: Delete this file. It is in the gitignore now but it was already committed.

File: package.json [hunk 2]
```diff
@@ -66,21 +77,26 @@
...
+    "docs": "sf-docs",
+    "format": "sf-format",
+    "lint": "sf-lint",
     "postpack": "shx rm -f oclif.manifest.json",
-    "prepack": "yarn run build && oclif-dev manifest",
-    "reformat": "prettier --config .prettierrc --write './*.{js,json,md}' './**/*.{ts,json,md}'",
-    "test": "nyc --extension .ts mocha --forbid-only \"test/**/*.test.ts\"",
     "posttest": "yarn lint",
+    "prepack": "sf-build",
+    "reformat": "prettier --config .prettierrc --write './*.{js,json,md}' './**/*.{ts,json,md}'",
```
Comment (amphro) [inline package.json:91]: I think this can be removed now in favor of just `format` right?

File: src/commands/hello/org.ts [hunk 3]
```diff
@@ -14,7 +14,7 @@ Messages.importMessagesDirectory(__dirname);
 
 // Load the specific messages for this file. Messages from @salesforce/command, @salesforce/core,
 // or any library that is using the messages framework can also be loaded this way.
-const messages = Messages.loadMessages('<REPLACE_ME>', 'org');
+const messages = Messages.loadMessages('@salesforce/plugin-template', 'org');
```
Comment (amphro) [inline src/commands/hello/org.ts:17]: I think this should stay `<REPLACE_ME>`

File: src/commands/hello/org.ts [hunk 4]
```diff
@@ -53,10 +53,10 @@ export default class Org extends SfdxCommand {
   protected static requiresProject = false;
 
   public async run(): Promise<AnyJson> {
-    const name = this.flags.name || 'world';
+    const name: string = this.flags.name || 'world';
 
     // this.org is guaranteed because requiresUsername=true, as opposed to supportsUsername
-    const conn = this.org!.getConnection();
```
Comment (amphro) [inline src/commands/hello/org.ts:59]: This do something like [this](https://github.com/forcedotcom/salesforcedx-apex/pull/64/files#diff-090d9fc625ac790f63e2a2b9a0ad93fdR33) to fix it.

File: src/commands/hello/org.ts [hunk 5]
```diff
@@ -66,14 +66,14 @@ export default class Org extends SfdxCommand {
     }
 
     // Query the org
+    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+    // @ts-ignore
```
Comment (amphro) [inline src/commands/hello/org.ts:70]: Why?

**Review by amphro — CHANGES_REQUESTED** [review]

File: src/commands/hello/org.ts (hunk 3 above)
Comment (WillieRuemmele) [reply src/commands/hello/org.ts:17]: if we keep it as <REPLACE_ME> the test won't pass and it won't compile. It's hard to use the dev-config and use this as a template with real code in it. We can just do `--no-verify` when committing and pushing to this repo too

**Review by amphro — APPROVED** [review]

**Pull request merged**